packages/contracts-bedrock/src/universal/StorageSetter.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { ISemver } from "src/universal/ISemver.sol";
+import { Storage } from "src/libraries/Storage.sol";
+
+/// @title StorageSetter
+/// @notice A simple contract that allows setting arbitrary storage slots.
+///         WARNING: this contract is not safe to be called by untrusted parties.
+///         It is only meant as an intermediate step during upgrades.
```

Comment:
Could enforce with an `onlyDeployer` modifier where the deployer is set during construction. But this is also just simple enough and clear enough to use like so.

Reply:
We can't use `onlyDeployer` unless the multisig deploys it, if we did only owner it would have to have an immutable owner. It is safe to be used the way that it is used at the moment